### PR TITLE
nominative month translation added to lt.js language

### PR DESCRIPTION
**Pull request by ignasne**

### CODE REVIEW

**Comment by ichernev** [comment]
@mmozuras what do you think?

@ignasne you need to change the unit tests and possibly add more to test the new functionality.

File: src/locale/lt.js [hunk 1]
```diff
@@ -24,6 +24,16 @@ function translateSeconds(number, withoutSuffix, key, isFuture) {
         return isFuture ? 'kelių sekundžių' : 'kelias sekundes';
     }
 }
+function monthsCaseReplace(m, format) {
+    var months = {
+            'nominative': 'sausis_vasaris_kovas_balandis_gegužė_birželis_liepa_rugpjūtis_rugsėjis_spalis_lapkritis_gruodis'.split('_'),
+            'accusative': 'sausio_vasario_kovo_balandžio_gegužės_birželio_liepos_rugpjūčio_rugsėjo_spalio_lapkričio_gruodžio'.split('_')
+        },
+        nounCase = (/D[oD]?(\[[^\[\]]*\]|\s+)+MMMM?/).test(format) ?
```
Comment (mmozuras) [inline src/locale/lt.js]: @ignasne could you explain this regex?

Comment (ichernev) [inline src/locale/lt.js]: Matches `D|Do|DD`, something custom (`[abc]` OR whitespace), then `MMMM`. I think the main thing is `D` followed by `MMM`.

**Comment by ichernev** [comment]
Fixed tests and merged in 4e3973f9fb4ad99808d9a439bec75f46715cea20

**Pull request closed without merging**